source/cli-translate.js

```diff
@@ -0,0 +1,128 @@
+var imagemagick_cli_lists = null;
+
+function addSelectOptions(node)
+{
+    var selectCliLanguageNode = document.createElement("select");
+    selectCliLanguageNode.className += "btn text-dark bg-light mx-4";
```

Comment:
I chose to match the existing class of the <pre> tag for aesthetic reasons. This could be changed to pull the classes from the <pre> instead if necessary.